packages/ng-schematics/src/ng-new/index_spec.ts

```diff
@@ -17,17 +17,6 @@ describe("igniteui-angular-schematics", () => {
 		const runner = new SchematicTestRunner("schematics", collectionPath);
 		const myTree = Tree.empty();
 		const workingDirectory = "my-test-project";
-		const mockLibrary = { getProject: jasmine.createSpy("getProject").and.returnValue(true), themes: ["custom"] };
```

Comment:
Since a name is provided and `defaults` is `true` by default, there is no longer a need for a mock of the `promptSession`, I agree.
We should add more tests to better cover the scenario though - calling w/o a name, calling w/ `defaults === false` and a combination of the different options to make sure it properly prompts for them (could be done in a separate PR IMO).